hydrosdk/servable.py

```diff
@@ -7,32 +7,30 @@
 from .exceptions import ServableException
 from .image import DockerImage
 from .model import Model
+from .predictor import Predictable
 
 
 class ServableStatus(Enum):
     """
     # TODO add more information about statuses
-    Servable can be in one of four states:
-        1. STARTING -
-        2. SERVING -
-        3. NOT_SERVING -
-        4. NOT_AVAILABLE -
+    Servable can be in one of four states
     """
     STARTING = 2
+    UNKNOWN = 2
```

Comment:
📝 Could you please put a comment that this is not a typo, but a fix for incorrect backend status

Reply:
Deleted